internal/util/copy.go

```diff
@@ -66,7 +66,23 @@ func CopyIndexFiles(logger *log.CmdLogger, src, dest string) error {
 	}
 
 	opt := copy.Options{
-		Skip: func(_ os.FileInfo, src, _ string) (bool, error) {
+		Skip: func(info os.FileInfo, src, _ string) (bool, error) {
+			logger.Debugf("Checking source copy path: %s", src)
+
+			if err != nil {
+				return true, fmt.Errorf(
+					"failed to create CouchDB index definitions from folder %s: %w",
+					src,
+					err,
+				)
+			}
+
+			if info.IsDir() {
+				return skipFolder(logger, indexSrcDir, src)
```

Comment:
I think it would be better to check for any errors here instead of just passing them on, e.g.

```
skip, err := skipFolder(logger, indexSrcDir, src)
if err != nil {
  return skip, fmt.Errorf(
			"<suitable error message with strings for dir/src being checked>: %w",
			...,
			err,
		)
}
```